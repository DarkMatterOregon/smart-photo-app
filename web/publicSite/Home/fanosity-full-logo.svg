<?xml version="1.0" encoding="utf-8"?>
<!-- Generator: Adobe Illustrator 23.0.2, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
	 viewBox="0 0 265.83 49.17" style="enable-background:new 0 0 265.83 49.17;" xml:space="preserve">
<style type="text/css">
	.st0{fill-rule:evenodd;clip-rule:evenodd;fill:#030512;}
	.st1{fill:#C99513;}
	.st2{fill:#030512;}
</style>
<g id="Guides_For_Artboard">
</g>
<g id="Layer_1">
	<path class="st0" d="M127.33,7.67c3.61,0.71,6.81,2.4,9.61,5.07c3.78,3.6,5.84,7.99,6.19,13.16c0.04,0.46,0.05,0.93,0.05,1.42
		c0,2.99-0.54,5.73-1.61,8.23c-0.11-0.12-0.22-0.25-0.33-0.37c-1.47-1.68-2.54-3.39-3.44-5.43c-1.42-3.2-2.42-5.86-3.44-9.16
		c-0.41-1.32-0.98-2.51-1.63-3.73c-1.53-2.87-3.27-5.71-4.94-8.5C127.63,8.12,127.48,7.88,127.33,7.67L127.33,7.67z M124.12,47.41
		l-0.02,0c-0.31,0-0.61,0-0.89,0c-5.49,0-10.19-1.82-14.11-5.45c-3.1-2.9-5.05-6.35-5.86-10.36l0.71,0.5
		c0.7,0.49,1.27,0.89,1.79,1.2c-2.04,3.61,2.33,5.95,5.09,7.12c1.26,0.53,2.58,0.98,3.83,1.54c0.89,0.4,7.63,3.22,8.23,3.88
		c0.33,0.37,0.66,0.88,0.99,1.28L124.12,47.41L124.12,47.41z M103.2,31.43c-0.17-0.87-0.28-1.77-0.35-2.69
		c-0.04-0.45-0.05-0.91-0.05-1.36c0-0.72,0.03-1.42,0.1-2.11c0.23,0.48,0.53,0.92,0.8,1.39L103.2,31.43L103.2,31.43z M103.87,20.64
		c0.22-0.65,0.48-1.29,0.77-1.91c0.03,0.36,0.09,0.71,0.17,1.04l-0.3,0.04l-0.41,0.53L103.87,20.64z"/>
	<path class="st0" d="M113.37,8.99l0.45-0.46l0.57,0.8l0.16,1.23l0.46-0.07c0,0,0,0.01,0,0.01l-0.12,0.28l0.62,0.63l-0.31,0.31
		l-0.86-1.28v2.3h-3.96V10.2h3.8L113.37,8.99L113.37,8.99z M110.32,14.43h7.04v7.69h-1.76c0.2-0.15,0.36-0.34,0.47-0.59
		c0.35-0.81-0.12-1.98-0.38-2.77l-0.07-0.21l-0.2-0.08c-0.15-0.06-0.29-0.11-0.44-0.16c-0.14-0.09-0.28-0.19-0.42-0.29
		c-0.02-0.01-0.05-0.03-0.07-0.04c-0.07-0.19-0.16-0.24-0.3-0.38l-0.07-0.07l-2.53-1.02l-1.26-0.02v-0.36
		c0.24-0.35,0.37-0.74,0.27-1.17l-0.02-0.09l-0.05-0.07c-0.07-0.1-0.14-0.2-0.2-0.31V14.43L110.32,14.43z M110.17,13.16h7.19v0.89
		h-7.19V13.16L110.17,13.16z"/>
	<g id="XMLID_1_">
		<g>
			<path class="st1" d="M111.96,7.65c0.25-1.01,0.02-2.12-0.39-3.16c-0.43,1.46-0.65,3.07-0.1,4.3
				C111.69,8.45,111.86,8.07,111.96,7.65z M111.38,0c2.43,3.73,2.86,6.99,0,9.46C109.29,7.64,109.37,4.43,111.38,0z"/>
		</g>
	</g>
	<path class="st0" d="M117.65,36.31l-3.02,1.22c-3.71,0.92-2.12,0.38-2.57,0.04L117.65,36.31L117.65,36.31z M108.19,21.86
		c1.34,0.13,4.41,0.54,5.71,0.36c2.45-0.33,1.94-1.49,1.34-3.31c-0.77-0.3-0.17,0.02-0.95-0.49c-0.05-0.04-0.2-0.09-0.24-0.23
		c-0.04-0.14-0.14-0.17-0.22-0.26l-2.35-0.94l-2.85-0.04c0,0,1.72-0.87,1.5-1.86c-0.79-1.13,0.33-0.29-1.48-0.54
		c-1.7,0.89-2.47,0.61-2.47,3.34C106.2,20.03,107.01,21.09,108.19,21.86L108.19,21.86z M105.36,26.39c-1.06-2.16-2.16-2.29,0-5.09
		c2.71-0.32,0.49,0.1,2.99,0.62c2.69,2.5,10.16-1.36,9.78,4.56C113.07,28.94,110.09,27.64,105.36,26.39L105.36,26.39z M114.95,10.03
		c1.09-0.16,0.92-0.36,0.49,0.65l1.91,1.94c4.47,1.9,4.19,0.52,6.07,5.12l-5.11-1.45c-0.18,1.54-0.28,5.26-0.04,6.76
		c0.59,3.76,0.74,1.02-0.1,4.88c1.64,0.88,1.71,0.76,1.77,3.25l-1.11-2.52c-3.37-0.7-3.97-0.29-7.36-0.27
		c-3.06,0.01-3.52-1.09-6.26-1.05l-0.36,3.47c2.57,1.82,2.89,2.15,6.64,1.65c3-0.69,4.47,0.02,7.84-1l-0.77,1
		c-0.05,0.07-0.17,0.38-0.21,0.31c-0.04-0.07-0.13,0.2-0.19,0.3c1.24,1.11,1.09,0.56,0.8,2.72l-1.33,0.54
		c0.07-0.09,0.18-0.2,0.22-0.26c0.04-0.06,0.16-0.19,0.2-0.27c0.06-0.11,0.28-0.5,0.33-0.62c0.38-1.02,0.13-0.33,0.14-1.29
		c-2.75-1.24-3.75-1.4-7.03-0.97c-2.09,0.15-3.37-0.44-4.3,1.05c-2.04,3.29,4.99,5.17,8.1,6.56c12.83,5.74,5.75,2.24,12.81,8.67
		l14.62-9.59c-1.16-2.32-3.9-3.54-6.4-9.18c-5.88-13.26-0.44-5.47-9.92-21.21c-1.49-2.48-2.46-2.17-5.06-1.27
		c-2.87,0.98-3.69,0.12-6.68,0.05L114.95,10.03z"/>
	<path class="st2" d="M0,8.17h26.94v4.76H5.26v11.79h19.06v4.63H5.26v17.66H0V8.17z"/>
	<path class="st2" d="M41.37,8.17h5.94l14.09,38.83h-5.76l-4.05-11.63H36.35l-4.21,11.63h-5.39L41.37,8.17z M50.1,31.09l-5.98-17.16
		L37.9,31.09H50.1z"/>
	<path class="st2" d="M66.58,8.17h6.2l19.37,31.38V8.17h5.02v38.83h-5.91L71.61,15.67v31.34h-5.02V8.17z"/>
	<path class="st2" d="M152.1,34.48c0.12,2.2,0.62,3.99,1.5,5.37c1.68,2.57,4.64,3.86,8.89,3.86c1.9,0,3.63-0.28,5.19-0.85
		c3.02-1.09,4.53-3.05,4.53-5.87c0-2.11-0.64-3.62-1.92-4.52c-1.3-0.88-3.33-1.65-6.09-2.3l-5.1-1.19
		c-3.33-0.78-5.68-1.63-7.07-2.56c-2.39-1.62-3.59-4.04-3.59-7.27c0-3.49,1.17-6.35,3.52-8.59c2.35-2.24,5.68-3.36,9.98-3.36
		c3.96,0,7.33,0.98,10.1,2.95c2.77,1.97,4.15,5.11,4.15,9.42h-4.94c-0.25-2.08-0.79-3.67-1.62-4.79c-1.54-2.03-4.15-3.04-7.83-3.04
		c-2.97,0-5.11,0.65-6.41,1.96c-1.3,1.3-1.95,2.82-1.95,4.55c0,1.9,0.77,3.3,2.31,4.18c1.01,0.56,3.29,1.27,6.84,2.11l5.28,1.24
		c2.54,0.6,4.51,1.42,5.89,2.46c2.39,1.82,3.59,4.45,3.59,7.9c0,4.3-1.53,7.38-4.58,9.23c-3.05,1.85-6.6,2.78-10.63,2.78
		c-4.71,0-8.4-1.23-11.06-3.7c-2.67-2.45-3.97-5.77-3.92-9.97H152.1z"/>
	<path class="st2" d="M186.39,8.17h5.31v38.83h-5.31V8.17z"/>
	<path class="st2" d="M228.91,8.17v4.63h-13.08v34.21h-5.31V12.8h-13.09V8.17H228.91z"/>
	<path class="st2" d="M231.18,8.17h6.13l11.16,18.66l11.16-18.66h6.19l-14.71,23.18v15.65h-5.26V31.36L231.18,8.17z"/>
</g>
</svg>
