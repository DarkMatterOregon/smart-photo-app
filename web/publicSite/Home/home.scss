@import "./meyer-reset";

#home-lander {
  font-family: 'Roboto', sans-serif;
  background: url('./concert-background.jpg') no-repeat center center fixed;
  -webkit-background-size: cover;
  -moz-background-size: cover;
  -o-background-size: cover;
  background-size: cover;
  width: 100%;
  height:  100vh;

  display: flex;
  flex-direction: column;
  justify-content: space-evenly;

  .headline-section {
    flex: 1;
    display: flex;
    flex-direction: column;
    align-items: center;

    padding-top: 150px;

    .full-logo {
      width: 500px;
    }

    h1 {
      color: darken(#cb8a23, 10%);
      font-size: 34px;
      line-height: 50px;
      font-weight: bold;
    }
  }

  .app-download-links {
    flex: 1;
    display: flex;
    flex-direction: row;
    justify-content: center;
    margin-top: 100px;

    .download-link {
      margin: 10px;
      width: 159px;

      .swap-on-hover {
        position: relative;
        margin:  0 auto;
        max-width: 159px;

        img {
          position: absolute;
          top: 0;
          left: 0;
          overflow: hidden;
          width: 159px;
          height: 55px;
        }

        .swap-on-hover__front-image {
          z-index: 9999;
          transition: opacity .2s linear;
          cursor: pointer;
        }
      }

      .swap-on-hover:hover > .swap-on-hover__front-image {
        opacity: 0;
      }
    }


  }
}


@media only screen and (max-width: 700px) {
  #home-lander {
    .headline-section {
      .full-logo {
        width: 80%;
      }
      h1 {
        font-size: 28px;
      }
    }
  }
}
