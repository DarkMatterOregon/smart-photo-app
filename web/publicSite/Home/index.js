import React, { Component } from 'react';
import fanosityLogo from './fanosity-full-logo.svg';
import googlePlay from './google-play.png';
import appStore from './app-store.png';
// import comingSoon from './coming-soon.png';

import './home.scss';

const APP_STORE = 'https://itunes.apple.com/us/app/fanosity/id972417667?mt=8';
const GOOGLE_PLAY = 'https://play.google.com/store/apps/details?id=com.fanosity';

export default class Home extends Component {
    render() {
        return (
            <div id="home-lander">
                <div className="headline-section">
                    <img src={fanosityLogo} className="full-logo"/>
                    <h1>The App for the Show</h1>
                </div>
                <div className="app-download-links">
                    <div className="download-link">
                        <a href={GOOGLE_PLAY}>
                            <img src={googlePlay} className="swap-on-hover__front-image" />
                        </a>
                    </div>
                    <div className="download-link">
                        <a href={APP_STORE}>
                            <img src={appStore} className="swap-on-hover__front-image" />
                        </a>
                    </div>
                </div>
            </div>
        );
    }
}

/*
For Flipping On Hover:
<figure className="swap-on-hover">
    <img src={appStore} className="swap-on-hover__front-image" />
    <img src={comingSoon} className="swap-on-hover__back-image" />
</figure>
 */
