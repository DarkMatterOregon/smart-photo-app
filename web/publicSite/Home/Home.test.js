import React from 'react';
import renderer from 'react-test-renderer';
import Home from './index';

describe('<Home />', () => {
    it('renders correctly', () => {
        const home = renderer.create(<Home />).toJSON();
        expect(home).toMatchSnapshot();
    });
});
