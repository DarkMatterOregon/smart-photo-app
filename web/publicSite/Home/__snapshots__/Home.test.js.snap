// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Home /> renders correctly 1`] = `
<div
  id="home-lander"
>
  <div
    className="headline-section"
  >
    <img
      className="full-logo"
      src="Test File Stub"
    />
    <h1>
      The App for the Show
    </h1>
  </div>
  <div
    className="app-download-links"
  >
    <div
      className="download-link"
    >
      <figure
        className="swap-on-hover"
      >
        <img
          className="swap-on-hover__front-image"
          src="Test File Stub"
        />
        <img
          className="swap-on-hover__back-image"
          src="Test File Stub"
        />
      </figure>
    </div>
    <div
      className="download-link"
    >
      <figure
        className="swap-on-hover"
      >
        <img
          className="swap-on-hover__front-image"
          src="Test File Stub"
        />
        <img
          className="swap-on-hover__back-image"
          src="Test File Stub"
        />
      </figure>
    </div>
  </div>
</div>
`;
